tests/regression/openconfig/openconfig_test.py

```diff
@@ -588,6 +604,32 @@ def testMixedDaddr(self):
         self.naming.GetNetAddr.assert_called_once_with('CORP_EXTERNAL')
         print(acl)
 
+    @capture.stdout
+    def testTcpEstablished(self):
+        self.naming.GetServiceByProto.return_value = ['80']
```

Comment:
@XioNoX I think you might have previously set a return value for GetNetAddr in this test but later removed it. I think you need to add it back. This suggested change fixes the test.

```suggestion
        self.naming.GetServiceByProto.return_value = ['80']
        self.naming.GetNetAddr.return_value = TEST_IPS
```

Reply:
Thanks! Looks like that was it. I was looking for issues in the generator and not in the test.